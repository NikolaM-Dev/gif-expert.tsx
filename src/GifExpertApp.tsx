interface IProps {}

export const GifExpertApp = ({}: IProps): JSX.Element => {
  return (
    <>
      <h1>GifExpertApp</h1>
    </>
  );
};
