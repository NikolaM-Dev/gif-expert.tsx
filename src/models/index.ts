export * from './ghipy-response.model';
export * from './gif.model';
