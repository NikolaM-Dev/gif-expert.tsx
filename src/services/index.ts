export * from './get-gifs.service';
