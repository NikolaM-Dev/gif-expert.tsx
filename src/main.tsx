import React from 'react';
import { createRoot } from 'react-dom/client';

import { GifExpertApp } from './GifExpertApp';

import './styles.css';

createRoot(document.getElementById('root') as HTMLElement).render(
  <React.StrictMode>
    <GifExpertApp />
  </React.StrictMode>,
);
