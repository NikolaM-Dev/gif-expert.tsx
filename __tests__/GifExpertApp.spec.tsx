import { render } from '@testing-library/react';

import { GifExpertApp } from '../src/GifExpertApp';

describe('Tests on <GifExpertApp />', () => {
  it('should to match with snapshot', () => {
    const { container } = render(<GifExpertApp />);

    expect(container).toMatchSnapshot();
  });
});
