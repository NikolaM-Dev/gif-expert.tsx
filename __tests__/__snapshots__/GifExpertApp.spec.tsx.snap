// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests on <GifExpertApp /> should to match with snapshot 1`] = `
<div>
  <h1>
    GifExpertApp
  </h1>
  <form
    aria-label="form"
  >
    <input
      placeholder="Search Gifs"
      type="text"
      value=""
    />
  </form>
  <h2>
    One Punch
  </h2>
  <p>
    Is Loading...
  </p>
  <section
    class="card-grid"
  />
</div>
`;
